recipes/wdltools/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "wdlTools" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/dnanexus-rnd/wdlTools/archive/{{ version }}.tar.gz
+  sha256: 115fd4ce8ae76e2963b284617bc14c72f494b11a4bef51c67baabd95d2a01974
+
+build:
+  skip: true  # [not unix]
+  number: 0
```

Comment:
Can this be `noarch: generic`? CC @mariusvniekerk

Reply:
Given the structure of the entrypoint no.